Lib/sqlite3/__main__.py

```diff
@@ -0,0 +1,91 @@
+import sqlite3
+import sys
+
+from argparse import ArgumentParser
+from code import InteractiveConsole
+from textwrap import dedent
+
+
+class SqliteInteractiveConsole(InteractiveConsole):
+
+    def __init__(self, database):
+        super().__init__()
+        self._con = sqlite3.connect(database, isolation_level=None)
+        self._cur = self._con.cursor()
+
+    def __del__(self):
+        self._con.close()
+
+    def runsql(self, sql):
+        if sqlite3.complete_statement(sql):
+            try:
+                self._cur.execute(sql)
+                rows = self._cur.fetchall()
+                for row in rows:
+                    print(row)
+            except sqlite3.Error as e:
+                print(f"{e.sqlite_errorname}: {e}")
+            return False
+        return True
+
+    def runpy(self, source, filename="<input>", symbol="single"):
+        code = self.compile(source, filename, symbol)
+        self.runcode(code)
+        return False
+
+    def printhelp(self, ignored):
+        print("Enter SQL code and press enter.")
+
+    def runsource(self, source, filename="<input>", symbol="single"):
+        keywords = {
+            "version": lambda x: print(f"{sqlite3.sqlite_version}"),
+            "help": self.printhelp,
+            "quit()": self.runpy,
+            "quit": self.runpy,
```

Comment:
> Isn't it an overkill? Just call os.exit() or raise SystemExit on "quit".

Good point; I'll fix that.

> BTW, the sqlite3 CLI uses commands starting with a dot: ".help", ".quit".

True. We could of course mimic that. I have no preference here, but I guess it will be nice for users if there is some familiarity with the sqlite3 cli.